src/assets/theme/sizes.tsx

```diff
@@ -1,8 +1,12 @@
+//TODO (Chelsi): rename these? 600px is generally used as our mobile breakpoint
+
```

Comment:
Maybe we should customize `md` to be 800px and keep `sm` with the default value (600px), we can discuss this later!